test/spec/ol/style/iconimagecache.test.js

```diff
@@ -39,22 +39,22 @@ describe('ol.style.IconImageCache', function() {
       expect(cache.cacheSize_).to.eql(4);
 
       src = '4';
-      iconImage = new ol.style.IconImage(src, null);
+      iconImage = new ol.style.IconImage(null, src);
```

Comment:
In https://github.com/openlayers/openlayers/commit/4981cde729d8f749cdb87fe5312ea0d1a780b23a#diff-7004065dfacb69406f951aa64d6e2c55R290, the `IconImage` constructor (then `IconImage_`) was given a new first argument (`img`).  But these tests were not updated.  Since then the tests have been creating icons with a string as the `image_` member.  I haven't dug into why they have been passing since then, but they start failing when we run the tests after using the ES modules transform.  Correcting the arguments to the constructor makes these tests pass again.